pkg/spyglass/lenses/buildlog/lens_test.go

```diff
@@ -61,6 +62,14 @@ func TestGetConfig(t *testing.T) {
 				}
 				return d
 			}(),
+		}, {
```

Comment:
nit: Add test case verifying what the iframe permissions are if none are specified.

Reply:
Added test @michelle192837 , good nit 👍